_posts/2022-05-16-zarr-2.12.0a1-release.markdown

```diff
@@ -6,4 +6,76 @@ categories: blog
 permalink: /release-2-12-0-a1/
 ---
 
-Updating soon...
\ No newline at end of file
+Pre-release version `2.12.0a1` of the [Zarr Python package](https://github.com/zarr-developers/zarr-python) has been released! 🎉
+
+This blog post aims to overview new features, especially newly added support for reading and writing to Zarr V3 arrays. Zarr V3 is the upcoming evolved format for storing N-dimensional chunked compressed arrays, which is succeeded by Zarr V2. This blog also highlights other enhancements like creating `FSStore` from an existing fsspec filesystem, performance improvement for Zarr arrays when appending data to S3, bug fixes, documentation and a maintenance fix.
+
+## Add suport for reading and writing Zarr V3
+
+This version provides the infrastructure for reading and writing the upcoming V3 spec of the Zarr format. The new `zarr._store.v3` package has the necessary classes and functions for evaluating Zarr V3. Since the design is not finalised, the classes and functions are not automatically imported into the regular Zarr namespace. However, setting the `ZARR_V3_API_AVAILABLE` environment will activate them.
+
+The pre-release can be installed via: `pip install —pre zarr`.
```

Comment:
This might could use a small piece of code that generates a v3 fileset and shows what it looks like, e.g. with `tree`.